<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomTour</class>
 <widget class="QMainWindow" name="CustomTour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back to Main Menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Plan a vacation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>611</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Stadium Selection</string>
    </property>
    <widget class="QPushButton" name="pushButton_add">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add this stadium</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Starting Stadium&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Selected Stadiums&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Available Stadiums&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_reset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Reset stadiums</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_selected">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>361</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <column>
      <property name="text">
       <string>Stadium</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Distance</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_dist">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>370</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Total Distance: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_distNum">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>370</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_stadiums">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Order to Visit</string>
     </property>
     <widget class="QRadioButton" name="radioButton_selected">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Selected Order</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_short">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Shortest Order</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_calculateTrip">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate Trip</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_error">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_souvenirs">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buy Souvenirs</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
